modules/node/src/NodeAccessControlHandler.php

```diff
@@ -103,6 +138,45 @@ protected function checkAccess(EntityInterface $node, $operation, AccountInterfa
       return AccessResult::allowed()->cachePerPermissions()->cachePerUser()->addCacheableDependency($node);
     }
 
+    // Revision operations.
+    $revisionPermissionOperation = isset(static::REVISION_OPERATION_MAP[$operation]) ? static::REVISION_OPERATION_MAP[$operation][0] : NULL;
```

Comment:
do we want named keys instead of 0 and 1 for understanding sake? or perhaps destructure using `list` so its clearer?

Reply:
yeh can use list